pkg/render/fluentd_test.go

```diff
@@ -70,6 +70,28 @@ var _ = Describe("Tigera Secure Fluentd rendering tests", func() {
 			ExpectResource(resources[i], expectedRes.name, expectedRes.ns, expectedRes.group, expectedRes.version, expectedRes.kind)
 			i++
 		}
+
+		ds := resources[1].(*apps.DaemonSet)
+		envs := ds.Spec.Template.Spec.Containers[0].Env
+
+		expectedEnvs := []corev1.EnvVar{
+			{Name: "FLUENT_UID", Value: "0"},
+			{Name: "FLOW_LOG_FILE", Value: "/var/log/calico/flowlogs/flows.log"},
+			{Name: "DNS_LOG_FILE", Value: "/var/log/calico/dnslogs/dns.log"},
+			{Name: "FLUENTD_ES_SECURE", Value: "true"},
+			{
+				Name: "NODENAME",
+				ValueFrom: &corev1.EnvVarSource{
+					FieldRef: &corev1.ObjectFieldSelector{FieldPath: "spec.nodeName"},
+				},
+			},
+		}
+
+		for _, expected := range expectedEnvs {
+
+			Expect(envs).To(ContainElement(
+				expected))
```

Comment:
Very Minor: Formatting seems a bit off here. Was there a reason for the newline in line 91 and having the Expect statement split across two lines?

Reply:
Updated!